src/Symfony/Component/HttpClient/Retry/HttpStatusCodeDecider.php

```diff
@@ -11,25 +11,56 @@
 
 namespace Symfony\Component\HttpClient\Retry;
 
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+
 /**
  * Decides to retry the request when HTTP status codes belong to the given list of codes.
  *
  * @author Jérémy Derussé <jeremy@derusse.com>
  */
 final class HttpStatusCodeDecider implements RetryDeciderInterface
 {
+    public const METHOD_DECISION = [
```

Comment:
This could be private

Reply:
Hmm.. I thing, this could be helpfull when building the list of status codes, ie. If user want to handle a new statusCode for idempotent methods.

Maybe this could be renamed

Reply:
I'd suggest naming these IDEMPOTENT_METHODS and making them a simple list, no need for these to be keys